Core/src/main/java/org/tribuo/ensemble/WeightedEnsembleModel.java

```diff
@@ -45,16 +48,35 @@
 
     protected final EnsembleCombiner<T> combiner;
 
-    public WeightedEnsembleModel(String name, EnsembleModelProvenance description, ImmutableFeatureMap featureIDMap,
+    /**
+     * Constructs a weighted ensemble model.
```

Comment:
In what sense is the 'weighted' if the weights are uniform, or am i missing something?

Reply:
Oh yeah, this one isn't. I think I copy/pasted it from the one below and changed the relevant params. It's a little weird as `EnsembleModel` is abstract, and so in practice if you want an unweighted combination you need to use `WeightedEnsembleModel` and this constructor. It should have a better name, but renaming things breaks compatibility.

Reply:
Fixed.